edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/EdgeDeploymentController.cs

```diff
@@ -74,12 +74,10 @@ public async Task<EdgeDeploymentStatus> DeployModulesAsync(ModuleSet desiredModu
                         {
                             [KubernetesConstants.K8sEdgeModuleLabel] = moduleIdentities[module.Key].DeploymentName(),
                             [KubernetesConstants.K8sEdgeDeviceLabel] = KubeUtils.SanitizeLabelValue(this.resourceName.DeviceId),
-                            [KubernetesConstants.K8sEdgeHubNameLabel] = KubeUtils.SanitizeLabelValue(this.resourceName.Hostname)
                         });
                 var deviceOnlyLabels = new Dictionary<string, string>
                 {
                     [KubernetesConstants.K8sEdgeDeviceLabel] = KubeUtils.SanitizeLabelValue(this.resourceName.DeviceId),
-                    [KubernetesConstants.K8sEdgeHubNameLabel] = KubeUtils.SanitizeLabelValue(this.resourceName.Hostname)
```

Comment:
why remove this label?

Reply:
The hostnames could be larger than the allowed label size in some private IoT Hub scenarios. Since each device is placed in it own namespace, we don't really need the hostname label. See #3066